examples/fedper_example/client.py

```diff
@@ -10,26 +10,25 @@
 from torch.optim import Optimizer
 from torch.utils.data import DataLoader
 
-from examples.models.fedper_cnn import FedPerGlobalFeatureExtractor, FedPerLocalPredictionHead
-from fl4health.clients.moon_client import MoonClient
+from examples.models.sequential_split_models import (
+    SequentialGlobalFeatureExtractorMnist,
+    SequentialLocalPredictionHeadMnist,
+)
+from fl4health.clients.fedper_client import FedPerClient
 from fl4health.model_bases.fedper_base import FedPerModel
-from fl4health.parameter_exchange.layer_exchanger import FixedLayerExchanger
-from fl4health.parameter_exchange.parameter_exchanger_base import ParameterExchanger
 from fl4health.utils.load_data import load_mnist_data
 from fl4health.utils.metrics import Accuracy, Metric
 from fl4health.utils.sampler import MinorityLabelBasedSampler
 
 
-class MnistFedPerClient(MoonClient):
+class MnistFedPerClient(FedPerClient):
```

Comment:
Changing this client to no longer inherit from MoonClient. I liked the idea of mixing them, because they are so related, but it can be a little confusing. This also means that we can rip out the need to remember to specify the exchanger when using vanilla FedPer. We can still use the MoonClient to combine the idea with FedPer, just removing it from the examples to reduce confusion.